workshop 01 - Introduction to Quantum Computing/README.md

```diff
@@ -5,3 +5,14 @@
 A shallow dive into the world of Quantum Computing! We explore what Quantum can
 achieve, what makes it better than it's classical counterpart? After convincing
 you all, we have made a special hands-on experience session, tailored for you :)
+
+This workshop had talks given by our Contributors and the powerpoint presentations for the same are 
+stored as such 
+
+| Topic | Speakers |
```

Comment:
Did you plan on separating the numbers? If not, remove the leading `|`.

Reply:
Removing the leading | _pipe_